include/seastar/util/process.hh

```diff
@@ -87,6 +87,11 @@ public:
     ///
     /// \returns the exit status
     future<wait_status> wait();
+    /// Stop the process using SIGTERM
+    future<> terminate();
+    /// Force the process to exit using SIGKILL
+    future<> kill();
```

Comment:
Do those need to return futures?

Reply:
hi @avikivity ahh, right. thought it was syscall, and syscall could be slow in comparison to user space function calls. but yeah, `kill()` does not block. so updated accordingly.